packages/studio-base/src/players/RosbridgePlayer.ts

```diff
@@ -436,6 +448,137 @@ export default class RosbridgePlayer implements Player {
     }
   }
 
+  // Helper methods
+  public scale_back_int16(value: number, min: number, max: number): number {
+    if (min === max) { return min; } // Prevent division by zero
+    return (value / 65535) * (max - min) + min;
+  }
+
+  public _base64decode(base64: string): ArrayBuffer {
+    const binary_string = window.atob(base64);
+    const len = binary_string.length;
+    const bytes = new Uint8Array(len);
+    for (let i = 0; i < len; i++) {
+        bytes[i] = binary_string.charCodeAt(i);
+    }
+    return bytes.buffer;
+  }
+
+  public decode_LaserScan_msg (message: any): void {
```

Comment:
For all these helped methods please use camelCase, since that is the convention in typescript methods